elastio-stream-kafka/consumer.py

```diff
@@ -0,0 +1,62 @@
+from kafka import KafkaConsumer
+import json
+import os
+import argparse
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+backup_parser = argparse.ArgumentParser(
+    prog="Elastio stream kafka consumer"
+)
+backup_parser.add_argument("--topic_name", type=str, nargs="?", help="Enter Kafka topic name to backup.")
+backup_parser.add_argument("--brokers", type=str, nargs="+", help="Enter one or more Kafka brokers separated by spaces.")
+args = backup_parser.parse_args()
+bootstrap_servers = args.brokers
+topic_name = args.topic_name
+
+
+consumer = KafkaConsumer(
+    topic_name,
+    group_id='elastio-group',
+    bootstrap_servers=bootstrap_servers,
+    auto_offset_reset='earliest', # latest/earliest
+    enable_auto_commit=True,
+    auto_commit_interval_ms=1000, # 1s 
+    consumer_timeout_ms=10000, # 10s
+    api_version=(2,0,2)
```

Comment:
That leads to the fact that the script supports the limited range of Kafka versions, so it should be a prerequisite to the script.

Reply:
If I understood you well you talk about this line _api_version=(2,0,2)_.
In this line, I specify the version of SDK that I use to create consumer not Kafka's version.